plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/generator/SQLGenerator.java

```diff
@@ -122,10 +123,18 @@ public boolean isShowPartitionsDDL() {
         return showPartitionsDDL;
     }
 
+    public boolean isShowCastParams() {
+        return showCastParams;
+    }
+
     public void setShowPartitionsDDL(boolean showPartitionsDDL) {
         this.showPartitionsDDL = showPartitionsDDL;
     }
 
+    public void setShowCastParams(boolean showCastParams) {
+        this.showCastParams = showCastParams;
+    }
+
```

Comment:
Please keep get-set pairs together

```suggestion
    public void setShowPartitionsDDL(boolean showPartitionsDDL) {
        this.showPartitionsDDL = showPartitionsDDL;
    }

    public boolean isShowCastParams() {
        return showCastParams;
    }

    public void setShowCastParams(boolean showCastParams) {
        this.showCastParams = showCastParams;
    }
```